client/src/stories/Button.stories.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Button from '../components/General/Button';
```

Comment:
General as folder name in Capital letters?

Reply:
Good advice, i did it in all folders wrong :D